testsuite/functional-testsuite/pom.xml

```diff
@@ -66,6 +66,14 @@
       <artifactId>httpmime</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <dependency>
+      <groupId>org.sonatype.nexus.testsuite</groupId>
+      <artifactId>nexus-modern-testsuite</artifactId>
+      <version>${it.nexus.bundle.version}</version>
+      <scope>test</scope>
+      <type>test-jar</type>
```

Comment:
as mentioned on chat, would prefer to have one or more "testsupport" modules containing these assets rather than introduce a "test-jar" dependency